cypress/e2e/smokeTests/sharedTests/waf.ts

```diff
@@ -55,12 +55,12 @@ describe('Test AWS Managed Common Rule Set', () => {
     });
   });
   // EC2MetaDataSSRF_URIPATH 	Inspects for attempts to exfiltrate Amazon EC2 metadata from the request URI path.
-  it('Try a URL that SHOULD be blocked if EC2MetaDataSSRF_URIPATH were turned on', () => {
-    const uglyUrl = '/PARAM=127.0.0.1+-c+0%3B+cat+%2Fetc%2Fpasswd&DIAGNOSIS=PING';
-    cy.request({ url: uglyUrl, failOnStatusCode: false }).then((resp) => {
-      expect(resp.status).to.eq(403);
-    });
-  });
+  // it('Try a URL that SHOULD be blocked if EC2MetaDataSSRF_URIPATH were turned on', () => {
```

Comment:
If we don't expect to restore the test, we should just delete it. If we think we might restore it, then a explanation/comment would be useful.

Reply:
I tried going through the logs to find any violations of this rule since it change and wasn't able to find any. That was how I constructed the test in the first place. For now, I'll just remove the test since it's not immediately clear how one would construct a violating request.